### PR TITLE
タグの表示を追加

### LINKED ISSUE #20: リストにタグを表示する

### LINKED ISSUE #36: 詳細画面でのタグの表示
現時点ではストリングでkeyとvalueどちらも表示している
これをタグだけ表示したい

### PR DESCRIPTION
**Pull request by kkenya**
## 概要
- リストにタグを表示する close #20 
- 詳細画面のタグ表示をタグ名だけに変更する close #36 

## 技術的変更点
- JSX内でmapメソッドによりタグの配列からタグ名を取得し表示する

## ToDo
- [x] リストでのタグ表示
- [x] 詳細画面でのタグの表示を変更する

### CODE REVIEW

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-add-showTagsOnList

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-add-showTagsOnList

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-add-showTagsOnList

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by TatsuyaKitagawa** [comment]
大丈夫だと思います

File: AddListView.js [hunk 1]
```diff
@@ -4,83 +4,98 @@ import { StackNavigator } from 'react-navigation';
...
-                            </View>
-                        </TouchableOpacity >
-                    );
-                }}
-            />
-        );
-    }
+  render() {
+    const { navigate } = this.props.navigation;
+    showTags = this.renderTags;
```
Comment (TatsuyaKitagawa) [inline AddListView.js]: ”showTags”はなんのためにありますか？

Comment (kkenya) [reply AddListView.js]: 消し忘れてました

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-add-showTagsOnList

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kkenya** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@kkenya/BookSeeker-add-showTagsOnList

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by kkenya** [comment]
マージします

**Pull request merged**